docs/operator-guide/troubleshooting/troubleshoot-stages.md

```diff
@@ -0,0 +1,38 @@
+# Application Is Not Deployed
+
+## Problem
+
+In the stage details page, the application status is unknown. The `deploy` button desn't work when attempting to create a stage.
```

Comment:
... doesn't work when attempting ...

Reply:
can we provide some logs or example how we can check that problem with appproject?